### DISCUSSION TITLE
🐑 Sheep! 🐑  … and 🍭 Candy 🍭 ?!?! 🤤 🤤 🤤

### DISCUSSION THREAD

**Discussion by Blasta-alasta** [sunflower-land/sunflower-land#2239] [Ideas]
_“For the Bumpkin world some things are not always as they seem. Here Sheep actually grow a delicious Cotton Candy Wool*, that can be harvested and refined into high energy sweets!”_

_Some of the feedback I got on my last proposal for Cows was that is was quite complicated and might be off-putting to newer/less advanced players. I do believe that some gameplay should be more advanced, but also it also makes sense to introduce simpler game play mechanisms too. With this Proposal, Sheep can fill that need!_

![Sheep](https://user-images.githubusercontent.com/102267293/218590078-dac87f1d-4053-4c07-800d-a6bb21059d2b.png)

### 🐑  Sheep! 🐑  … and 🍭 Candy 🍭 ?!?!
_**Game Loop:**_

- Build **Sheep Shed**
- Purchase **Sheep** > Feed > Wait 48 Hours > Harvest **Wool** > Repeat
- Discover Rare **Mutant Sheep** along the way
- Wool Produced is **Cotton Candy Wool**
- Build **Candy Factory**
- Use **Cotton Candy Wool** & other Crops/Fruits to make delicious **Candy**
- Use **Candy** for **Time Limited Boosts**

_**Also:**_

- Complete **Candy Quests** for **Rare Bumpkin Items**
- Compete in the **Gummy Bear Race** to obtain **Rare SFTs**, which will you go for?

![sheep happy](https://user-images.githubusercontent.com/102267293/218590269-f11892f5-c577-4c75-9215-2a631f6494a0.png)![sheep happy](https://user-images.githubusercontent.com/102267293/218590279-e84db47e-552e-4a64-9660-18da570b622e.png)![sheep happy](https://user-images.githubusercontent.com/102267293/218590284-9747345c-4ae2-4473-bc19-b404c8fc5120.png)

### The Sheep Shed
_A Sheep Shed acts the same as a Hen House does for chickens, except players will be able to purchase their first Sheep._

**Building Requirements** (for 1st Sheep Shed)
- 100 Wood
- 100 Stone
- 10 Iron
- 9.75 SFL
- Level 7 Bumpkin
- 24 hours to complete
- 3 by 2 square space

**Purchasing Sheep**
- 5 SFL each
- Limit of 5 (per Sheep Shed)

### Sheep Harvesting & Feeding
_Every 48 hours Sheep need a good session with the sheep shearers to clear that thick coat of **Cotton Candy Wool**…._
- Similar to harvesting Chickens a Player will have to click their Sheep once the timer has completed, which will then drop **Cotton Candy Wool**. 
- Sheep will drop 1 **Cotton Candy Wool**, but there will be a 20% chance of an additional 1.
- There will also be a much smaller % chance of a **Mutant Potion** drop.
- Sheep will then require their feed : **1 Radish** (each) to restart the 48 hour timer._

![sheep happy](https://user-images.githubusercontent.com/102267293/218590269-f11892f5-c577-4c75-9215-2a631f6494a0.png) ![sheep ready](https://user-images.githubusercontent.com/102267293/218590545-16ab45d0-af26-410e-87e6-edf063125637.png) ![sheep hungry](https://user-images.githubusercontent.com/102267293/218590559-c4a05e9b-f100-4f59-8506-998561a2a241.png)

### Candy Factory
_Can’t make Candy without a Candy Factory…. And this one has a cute Candy Shop out front!_

**_Requirements_**
- 300 Wood
- 5 Gold
- 10 SFL
- Level 9 Bumpkin
- 24 hours to complete
- 4 by 3 square space

### Candy
_Bumpkins love Candy…. It really spikes their energy levels…. But after too much a Bumpkin will crash! That’s why smart Bumpkins will only consume 2 pieces a day!_

For Candy I have tried to go a different route than with Chickens and Cakes to try keep things fresh… instead of the produce being made into food for **EXP**, produce from Sheep (**Cotton Candy Wool**) will be used to make **Candy**.

**Candy** will have a different boosts that last for a certain amounts of time, once consumed by a Players Bumpkin.

**_Bumpkin Bubblegum_** 🍬
Boost: Crops planted while chewing give extra 5% yield (Last for 1 hour)

- 5 Cotton Candy Wool
- 45 Beetroots
- 100 Potatoes
- 2 Hours to Make

**_Easters Eggs_** 🍬
Boost: Cheer for your chickens and knock 1 hours off their current harvest time.

- 5 Cotton Candy Wool
- 5 Eggs
- 4 Hours to Make

**_Candy Apples_** 🍬
Boost: The Fruit-picker's Treat... Fruit trees planted in next 20 mins will have have 20% reduction in growing/replenishing time.

- 7.5 Cotton Candy Wool
- 10 Apples
- 4 Hours to Make

**_Berry Sweets_** 🍬
Boost: A Bumpkin Favourite -  Radishes planted in the next 20 min will give +25% on harvesting

- 7.5 Cotton Candy Wool
- 10 Blueberries
- 8 Hours to Make

**_Sugar Free Candy_** 🍬
Boost: A Healthy Treat to enjoy with Lunch or Dinner... +20% EXP Boost For 20 mins

- 10 Cotton Candy Wool
- 300 Sunflowers
- 10 Parsnips
- 12 Hours to Make

**_Hardboiled Lollies_** 🍬
Boost: Gives Bumpkin extra energy while mining… +0.05 to all nodes mined in the next 20mins

- 12.5 Cotton Candy Wool
- 10 Kale
- 20 Pumpkin
- 12 Hours to Make

![sheep happy](https://user-images.githubusercontent.com/102267293/218590269-f11892f5-c577-4c75-9215-2a631f6494a0.png)![sheep black](https://user-images.githubusercontent.com/102267293/218590793-096ee825-7fc0-41f0-97c8-81e39f26662b.png)![sheep happy](https://user-images.githubusercontent.com/102267293/218590284-9747345c-4ae2-4473-bc19-b404c8fc5120.png)

### Mutant Sheep
**Mutant Potion**
_RNG Chance that if Player has a **Yam** on their farm, that a Sheep will have a chance of dropping a Rare **Mutant Potion** as well as **Cotton Candy Wool**…
Players will use the **Mutant Potion** on one of their sheep (at the Cow Shed) and it will be transform into a **Mutant Sheep**, each with a **Special Boost**!_

Mutant Sheep work differently than Mutant Chickens in four ways:

1. They are not tradable (non-withdrawable)
2. They have to be created from a regular Sheep, from which they then take the place of that Sheep, and produce Cotton Candy Wool is the same way, but with the associated Boost it gets from being a Mutant.
3. They require another SFT (Yam) for the drop chances to be activated.
4. The associated Boost is given only to that particular Mutant Sheep.

**_NOTE:_** What this means is that if a player has 5 Sheep (and a Ram), and 1 of the sheep drops a Mutant, then the Player will have to decide when to use the Mutant Potion, and on which one of his Sheep.

Because these Mutants last forever, and are non-withdrawable, the chance of finding Mutant Potion is quite Rare. Then once applied to a Sheep the chances of each Mutant Sheep outcome is different.

**Mutant Sheep**

**_Magic Merino_** 🐑 (Black Sheep) (rare) 
_“This wool is extra lush”_ 
Boost: Produces +0.1 more wool

**_Lamb Chops_** 🐑 (Pink Sheep) (rare)
_“Oh great… a sheep that doesn’t stop singing”_
Boost: Requires **1 Pumpkins** instead of **1 Radish** for feeding

**_Rainbow Romney_** 🐑 (Rainbow Sheep) (rare)
_“Every Candy Factory needs one of these”_
Boost: If placed in close enough to a Players Candy Shop, Candy cooking time is reduced by 10%

**_Super Sheep_** 🐑 (super rare)
_“Just always has to outdo the rest”_ 
Boost: Produces 50% quicker

**_Gold Dipped Ewe_** 🐑  (ultra rare) 
_“Her hooves are dipped in ggggoooollldddddddd!”_
Has a 1/10 chance of dropping 0.15 Gold as well as Cotton Candy Wool (ultra rare)

![sheep ready](https://user-images.githubusercontent.com/102267293/218590948-db459897-e3bc-445c-bdca-dd5fb13cdebf.png)![sheep ready](https://user-images.githubusercontent.com/102267293/218590948-db459897-e3bc-445c-bdca-dd5fb13cdebf.png)![sheep ready](https://user-images.githubusercontent.com/102267293/218590948-db459897-e3bc-445c-bdca-dd5fb13cdebf.png)

### SFT/NFTs & Gummy Bear Race

When Sheep are released a similar race could be done as was done with the Fruit Release. Players will have strategise to decide which rare Items they will want to obtain. Gummy Bears can be crafted during these event at the Candy Factory. A player can Craft as many of these as they desire. They are Cosmetic (displayable on a Player farm), but also will be utilised to craft Rarer NFTs during the Race.

The idea is that a few different Limited Edition NFTs will be available, but Players will have to strategise for which ones they want, and in which order... and it might not be possible for 1 Player to go for all of them (as was/is with the Fruit Race).

_**Angry (Red) Gummy**_  🤤 (Unlimited - Candy Factory - Non-withdrawable)
Decoration only (can be used to craft Limited Edition NFTs)
**Requirements** - 5 Candy Apples

**_Dopey (Green) Gummy_** 🤤 (Unlimited - Candy Factory - Non-withdrawable)
Decoration only (can be used to craft Limited Edition NFTs)
**Requirements** - 5 Sugar-Free Candy

**_Clever (Blue) Gummy_** 🤤 (Unlimited - Candy Factory - Non-withdrawable)
Decoration only (can be used to craft Limited Edition NFTs)
**Requirements** - 8 Berry Sweets

**_Happy (Yellow) Gummy_** 🤤 (Unlimited - Candy Factory - Non-withdrawable)
Decoration only (can be used to craft Limited Edition NFTs)
**Requirements** - 5 Hardboiled Lollies

**_Ram_** (Unlimited - Helios - Non-withdrawable)
A Ram is an NFT that needs to be placed independently of the other Sheep. It is craft-able at Helios with an unlimited supply and is non-withdraw-able. A Ram will be needed to activate the chances of getting a Mutant potion.
Boost : Reduce Sheep Harvest Time by 10 Hours & Activates Mutant Potion Drop Chance
**Requirements**
5 Sugar Free Candy
5 Gold

_**Alfalfa Bush**_ 🤤 (Limited to 2100 - Goblin Retreat)
Boost : -0.2 to feeding requirement
**Requirements**
- 1 Clever Gummy Bears
- 1 Dopey Gummy Bears
- 10 Gold

_**Farm Pup**_ 🐶 (Limited to 2100 - Goblin Retreat)
Boost : +0.1 to wool harvests.
**Requirements**
- 1 Angry Gummy Bears
- 1 Happy Gummy Bears
- 10 Gold
![pup4 best2](https://user-images.githubusercontent.com/102267293/218602480-cd58ffb0-0e7e-49b3-923e-787794f6652b.png)  ![farm_dog](https://user-images.githubusercontent.com/102267293/218602220-5bbc4608-0bf5-4df6-80be-7aaff1971d6a.gif) ![pup4 best](https://user-images.githubusercontent.com/102267293/218602246-4c53a90a-fda4-4571-a69d-104dab9eaf97.png) ![pup2](https://user-images.githubusercontent.com/102267293/218602258-8464965b-5dd6-4da3-9cae-b1e1dcf05532.png)
**Note:** Miniature version of the Legacy Farm Dog, allowing new and old players to have a dog for their farms.

**_Candy Man_** 🍭 (Limited to 1400 - Goblin Retreat)
Boost : Sweeten’s every deal - Candy Boosts
**Requirements**
- 1 Angry Gummy
- 1 Dopey Gummy
- 1 Clever Gummy
- 1 Happy Gummy
- 25 Gold

**_Golden Chompers_** 🍬 (Bumpkin Mouth/Teeth - Limited to 500 - Goblin Retreat)
Boost : Allows bumpkin to consume twice his daily candy allowance
**Requirements**
- Rotten Candy Mouth (Bumpkin Item)
- 1 Angry Gummy
- 1 Dopey Gummy
- 1 Clever Gummy
- 1 Happy Gummy
- 50 Gold

### Quest Bumpkin Items:
_Along with the above NFTs, related Bumpkin Items could also be introduced alongside Cows, again as was done with the Fruit Quest. These items would be unlimited for a limited time._

**_Candy Pants_** 🍭 (Bumpkin pants - cosmetic)
Shear Sheep 20 times

_**Sheep Shearer**_ 🍭 (Bumpkin handheld - boost) 
Boost: Sheep produce 10% quicker.
Make 20 Candy

_**Candy Woollen Jersy**_ 🍭 (Bumpkin Shirt - cosmetic)
Buy 5 Gummies

_**Rotten Cotten Mouth**_ 🍭 (Bumpkin Mouth/Teeth - cosmetic)
Eat 100 Candy
**Note:** This item is cosmetic but is needed to craft the rarer Golden Chompers.

![sheep hungry](https://user-images.githubusercontent.com/102267293/218590994-726a044e-471d-4f99-a82f-bc1b981115f8.png)![sheep hungry](https://user-images.githubusercontent.com/102267293/218590994-726a044e-471d-4f99-a82f-bc1b981115f8.png)![sheep hungry](https://user-images.githubusercontent.com/102267293/218590994-726a044e-471d-4f99-a82f-bc1b981115f8.png)

### Exisiting Boosts

_There are already many exisiting Animal Boosts available to players. The following would also apply to sheep:_

Legacy Skill - Wrangler (Animals produce 10% more) / Increase chance of mutants)
Legacy Skill - Barn Manager (Animals produce 10% quicker) / Increase chance of mutants)
Bumpkin Skill - Stable Hand (Animals produce 10% quicker)
Bumpkin Skill - Free Range (Animals produce 10% more)
Bumpkin Skill - Horse Whisperer (Increase chance of mutants)
Bumpkin Skill - Buckaroo (Chance of double drops)
Bumpkin Skill - Rush Hour (Cook meals 10% quicker)

**Farm Dog** 
![farm_dog](https://user-images.githubusercontent.com/102267293/218601932-b5b25b26-33ec-4d36-9e94-827d0ba15c9e.gif)
 (Limited to 410 - Legacy Item)
Boost : Sheep produce 20%/30% more wool (+0.2/+0.3 Wool)

_Note:_ Obviously owners of a Farm Dog and/or Animal Legacy Skills will start the race with an advantage, but NFT numbers have been into account to reflect this advantage for those players.

![Sheep Dog Examples](https://user-images.githubusercontent.com/102267293/218601973-e33a68bd-778a-4e25-ae5c-d9a1764752d9.png)

*The idea that Wool can be turned into Candy comes from the thought of what a long term use for Sheep and Wool could be. Obviously is makes more sense for Wool to used to make Bumpkin Clothing, but there is only so much clothing Bumpkins can have in their wardrobes and there are already getting pretty full!

Expect any ideas, values or mechanisms to change if anything from this is introduced.

🐑🍭 🐑🍭 🐑 🍭🐑🍭 🐑🍭 🐑 🍭🐑🍭 🐑🍭 🐑 🍭

**Comment by TheDcol** [comment]
I like the various rewards/items you can craft.. and time limited boosts would also add a lot of strategy in the game
